api/src/services/teams.js

```diff
@@ -83,6 +88,45 @@ class OSMTeams {
     return rp(`${OSM_TEAMS_SERVICE}/api/teams/${id}`)
   }
 
+  /**
+   * Check if user has permission to edit a team. The user needs to be either an
+   * owner of the organization, or a moderator of the team. Note: this should be
+   * the same as osm-teams's own team:edit permission.
+   *
+   * @param id {number} The osm-teams team id
+   * @returns {Promise<boolean>}
+   * @async
+   */
+  async canEditTeam (id) {
+    try {
+      const [orgOpts, teamOpts] = await Promise.all([
+        this.addAuthorization({
+          method: 'GET',
+          uri: `${OSM_TEAMS_SERVICE}/api/organizations/${id}`,
```

Comment:
I think you mean for this `id` to be the environment variable `OSM_TEAMS_ORG_ID` and not the team id, but you should merge in the latest from develop into your branch so you can access that.

Reply:
👍 good catch